tests/template_test.go

```diff
@@ -113,3 +120,13 @@ func deleteIfNotFailed(e *ptesting.Environment) {
 		e.DeleteEnvironment()
 	}
 }
+
+func isWhiteListedTest(templateName string, whiteListPrefix []string) bool {
+	for _, whiteList := range whiteListPrefix {
+		if strings.Contains(templateName, whiteList) {
```

Comment:
Since you check on `Contains`, "prefix" sounds like a confusing name to me (it would be good for Starts With)